process/smartContract/hooks/blockChainHook.go

```diff
@@ -261,8 +265,12 @@ func (bh *BlockChainHookImpl) GetBlockhash(nonce uint64) ([]byte, error) {
 	return hash, nil
 }
 
-// LastNonce returns the nonce from from the last committed block
+// LastNonce returns the nonce from the last committed block
 func (bh *BlockChainHookImpl) LastNonce() uint64 {
+	if bh.flagDisableBlockchainOperations.IsSet() {
```

Comment:
Last"*" operations are needed and they are good

Reply:
done